docs/developing.md

```diff
@@ -11,7 +11,7 @@ On all platforms, the minimum requirements are:
 ### Windows
 
 - [Visual Studio 2022 (17.1)](https://visualstudio.microsoft.com/downloads/) or newer
-  - Workloads (with recommeded components):
+  - Workloads (with recommended components):
```

Comment:
We need some tool to capture typos like this one. @theletterf any suggestions?

Reply:
https://github.com/markdownlint/markdownlint perhaps?

Reply:
I am personally using [`misspell`](https://github.com/client9/misspell)

I also saw https://github.com/open-telemetry/opentelemetry-dotnet/pull/3127 which adds config for https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker which AFAIK under the hood uses https://github.com/streetsidesoftware/cspell

> https://github.com/markdownlint/markdownlint perhaps

This is just checking the markdown formatting.

We can add a `lint-docs` GitHub workflow issue. I am happy to work on it after the beta release as it would help maintaining the docs.

Reply:
Created https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/473